Makefile.riscv64

```diff
@@ -6,23 +6,20 @@ ifeq ($(CORE), x280)
 CCOMMON_OPT += -march=rv64imafdcv_zba_zbb_zfh_zvl512b -mabi=lp64d 
 FCOMMON_OPT += -march=rv64imafdcv_zba_zbb_zfh -mabi=lp64d -static
 endif
-ifeq ($(CORE), RISCV64_ZVL256B)
+RISCV64_OPT  = rv64imafdcv
```

Comment:
Since more than one option can be active, we need to start chaining compiler options for -march

Reply:
Makes sense, thanks.